td/tracking/templates/tracking/project_list.html

```diff
@@ -26,9 +26,8 @@
                 {% endblocktrans %}
             {% else %}
             <p>
-                {% url "account_login" as login_url %}
                 {% blocktrans %}
-                    <a href="{{ login_url }}" class="btn btn-default">Log In to Enter Project or Event</a>
+                    <a href="/account/login/?next=/tracking/" class="btn btn-default">Log In to Enter Project or Event</a>
                 {% endblocktrans %}
```

Comment:
Ah, okay. I didn't know how to add the `next` param to the `{{ login_url }}`. Now I do :)